src/components/WebexAvatar/WebexAvatar.jsx

```diff
@@ -26,16 +48,28 @@ export default function WebexAvatar({
   const type = displayStatus ? status : null;
 
   const cssClasses = webexComponentClasses('avatar', className);
+  const oldClasses = webexComponentClasses('old-avatar', className);
```

Comment:
the old avatar is never used with an optional class (search for "<Avatar" and "<WebexAvatar" in the project), and it also did not have a css file before, so I guess, to make it simple to remove the old implementation, it would be ok to just add `<Avatar className="wxc-old-avatar"` below